index.html

```diff
@@ -2501,22 +2501,22 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
 
         <ol>
           <li>
-            <p data-lang="en"> The base text contains one or more Han character. Rules for separating terms can be found in <cite>GB/T 16159—2012 Basic Rules of Hanyu Pinyin Orthography</cite>.</p>
+            <p data-lang="en">The base text contains one or more Han characters. Rules for separating terms can be found in <cite>GB/T 16159—2012 Basic Rules of Hanyu Pinyin Orthography</cite>.</p>
```

Comment:
`<cite>GB/T 16159—2012 Basic Rules of Hanyu Pinyin Orthography</cite>` → `<cite>Basic Rules of Hanyu Pinyin Orthography</cite> (GB/T 16159—2012)`